lib/sis_importer.rb

```diff
@@ -114,7 +114,7 @@ def fetch_course_attributes(class_xml)
         subject:             class_xml.xpath('Subject').text.strip,
         number:              class_xml.xpath('CatalogNbr').text.strip,
         description:         class_xml.xpath('Description').text.strip,
-        units:               class_xml.xpath('UnitsMin').text.strip, #TODO: Should this be min or max, or something else?
+        units:               class_xml.xpath('UnitsMin').text.strip, #TODO: Should this be min, max, or something else?
```

Comment:
Line is too long. [119/100]<br>Unnecessary spacing detected.<br>Missing space after #.